ports/atmel-samd/mpconfigport.mk

```diff
@@ -57,6 +57,7 @@ CIRCUITPY_ZLIB = 0
 
 ifeq ($(INTERNAL_FLASH_FILESYSTEM),1)
 CIRCUITPY_ONEWIREIO ?= 0
+CIRCUITPY_SAFEMODE_PY ?= 0
```

Comment:
Does this mean we don't have it on trinket m0?

Reply:
It doesn't fit on most M0 non-external-flash boards. On Trinket M0 it just barely fits. Other boards have more pins and push the builds over the edge.

Reply:
Hrm. I agree it'll likely grow with the new error messages. This feels like something that should be available everywhere. We can leave it disabled for now and enable it later when we figure out how.